trojsten/submit/templates/trojsten/submit/protocol.html

```diff
@@ -34,7 +34,9 @@ <h4>Výstup z kompilátora</h4>
                     {% if test.showDetails %}
                 <tr>
                 <td colspan="3"  style="padding:0">
-                        <div  class="collapse" id="collapse{{ forloop.counter0 }}"> {{ test.details|linebreaksbr }}</div>
+                    <div class="collapse" id="collapse{{ forloop.counter0 }}">
+                        <pre>{{ test.details }}</pre>
```

Comment:
uplne korektne by bolo:
```
<pre><samp>{{ test.details }}</samp></pre>
```
ale whatever.

Reply:
nikdy som o tomto elemente nepocul, ale ano, ked citam tie docs, tak to ma byt hentak